framework/src/play/mvc/Router.java

```diff
@@ -614,6 +622,26 @@ public static ActionDefinition reverse(String action, Map<String, Object> args)
         return matchingRoutes;
     }
 
+  /**
+   * Prioritize action routes based on active locale and Play.langs properties. Active lang has highest priority, then prioritized according to Play.langs order.
+   *
+   * @param matchingRoutes
+   */
+  private static void prioritizeActionRoutesBasedOnActiveLocale(List<Router.ActionRoute> matchingRoutes) {
+        if(matchingRoutes.size()==0) return;
+        final String locale = Lang.get();
+        if(StringUtils.isEmpty(locale)) return;
+        matchingRoutes.sort(new Comparator<ActionRoute>() {
```

Comment:
I run same build on my machine with ant -buildfile ./framework/build.xml test
And build is succesfull.

I use anonymous comparator

`sortedMatchingRoutes.sort(new Comparator<ActionRoute>() {
            @Override
            public int compare(ActionRoute ar1, ActionRoute ar2) {
                if(locale.equals(ar1.route.locale)) return -1;
                if(locale.equals(ar2.route.locale)) return 1;
                return Integer.compare(Play.langs.indexOf(ar1.route.locale), Play.langs.indexOf(ar2.route.locale));
            }
        });`